### PR TITLE
Merge dev-integrate -> main

### PR DESCRIPTION
**Pull request by calebofearth**

### CODE REVIEW

File: src/soc_ifc/rtl/soc_ifc_external_reg.rdl [hunk 1]
```diff
@@ -226,6 +264,12 @@ reg {
 reg {
     name = "DEBUG & MANUF SERVICE REG";
     desc = "JTAG in debug/manuf mode or SOC can write to this register for ROM/FW defined skips or services; ROM/FW maintains the defintion of these bits.
+            [br]Field decode:
+            [br] [0]     MFG_FLAG_GEN_IDEV_CSR: Enable bit for Caliptra to generate an IDEV CSR
+            [br] [15:1]  MFG_FLAG_RSVD
```
Comment (nquarton) [inline src/soc_ifc/rtl/soc_ifc_external_reg.rdl:269]: There was a recent change to assign bit 1 to " Random Number Generator Unavailable". Would it be possible to add this here as well?

Comment (calebofearth) [reply src/soc_ifc/rtl/soc_ifc_external_reg.rdl:269]: Yes. Are bit assignments for this register expected to continue evolving prior to 1.0?

**Review by Nitsirks — APPROVED** [review]

File: src/soc_ifc/rtl/soc_ifc_external_reg.rdl (hunk 1 above)
Comment (calebofearth) [reply src/soc_ifc/rtl/soc_ifc_external_reg.rdl]: Resolved by #283

Comment (nquarton) [reply src/soc_ifc/rtl/soc_ifc_external_reg.rdl]: Thanks for adding that! I wouldn't say more changes are expected. But I can't say for sure no other bit assignments would be needed.

**Review by mcockrell-google — APPROVED** [review]

**Review by howardtr — APPROVED** [review]

**Pull request merged**